pkg/deploy/service_test.go

```diff
@@ -171,6 +179,52 @@ func (suite *ServiceTestSuite) TestGetManifestsFromPathSuccess() {
 	}
 }
 
+func (suite *ServiceTestSuite) TestGetManifestsFromGithubPathSuccess() {
+	skipCI(suite.T())
```

Comment:
Messing with GITHUB_WORKSPACE in github doesn't go well. Skip this test in that env.